Source/WebCore/platform/unix/SharedMemoryUnix.cpp

```diff
@@ -28,6 +28,10 @@
 #include "config.h"
 #include "SharedMemory.h"
 
+#if defined(__ANDROID__) || defined(ANDROID)
```

Comment:
We use this in other files already, could we add a `OS(ANDROID)` definition?

Reply:
I have made a separate patch to add `OS(ANDROID)` and change the existing preprocessor guards: https://github.com/WebKit/WebKit/pull/43460 &mdash; I'll change the guards in this and other upcoming PRs accordingly.